blackbox/requirements.txt

```diff
@@ -39,3 +35,18 @@ zc.customdoctests==1.0.1
 zope.exceptions==4.1.0
 zope.interface==4.4.2
 zope.testrunner==4.7.0
+
+# the next section should exact copy of the hosted RTD environment (please
+# update if you notice that this is not the case)
+
+alabaster>=0.7,<0.8,!=0.7.5
+commonmark==0.5.4
+docutils==0.13.1
+mock==1.0.1
+pillow==2.6.1
+Pygments==2.2.0
+readthedocs-sphinx-ext<0.6
```

Comment:
Why do we need `readthedocs-sphinx-ext` and `sphinx-rtd-theme` for local development?

Reply:
Ok, I understand why we would need the readthedocs-sphinx-extension, but the sphinx-rtd-theme is never used, not even on readthedocs, since we have our own theme. that's the whole point.

Reply:
@chaudum pushed a fixup. does this look ok now?